************* Module eCloud.ecloud
ecloud.py:71:1: W0511: TODO: move to new Util class (fixme)
ecloud.py:81:78: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
ecloud.py:82:20: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
ecloud.py:83:24: C0103: Variable name "l" doesn't conform to snake_case naming style (invalid-name)
ecloud.py:84:24: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
ecloud.py:89:35: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud.py:89:28: C0103: Variable name "me" doesn't conform to snake_case naming style (invalid-name)
ecloud.py:97:24: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
ecloud.py:102:35: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud.py:102:28: C0103: Variable name "ce" doesn't conform to snake_case naming style (invalid-name)
************* Module eCloud.ecloud.globals
ecloud/globals.py:10:1: W0511: TODO: make a Consts class (fixme)
ecloud/globals.py:11:1: W0511: TODO: make a Globals class (mutable) (fixme)
************* Module eCloud.ecloud.logging_ecloud
ecloud/logging_ecloud.py:7:0: W0311: Bad indentation. Found 3 spaces, expected 4 (bad-indentation)
ecloud/logging_ecloud.py:45:0: C0305: Trailing newlines (trailing-newlines)
ecloud/logging_ecloud.py:1:0: C0114: Missing module docstring (missing-module-docstring)
************* Module eCloud.ecloud.opencda_carla
ecloud/opencda_carla.py:24:8: C0103: Attribute name "x" doesn't conform to snake_case naming style (invalid-name)
ecloud/opencda_carla.py:24:16: C0103: Attribute name "y" doesn't conform to snake_case naming style (invalid-name)
ecloud/opencda_carla.py:24:24: C0103: Attribute name "z" doesn't conform to snake_case naming style (invalid-name)
ecloud/opencda_carla.py:23:23: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)
ecloud/opencda_carla.py:23:37: C0103: Argument name "y" doesn't conform to snake_case naming style (invalid-name)
ecloud/opencda_carla.py:23:51: C0103: Argument name "z" doesn't conform to snake_case naming style (invalid-name)
ecloud/opencda_carla.py:36:8: W0621: Redefining name 'Vector3D' from outer scope (line 11) (redefined-outer-name)
ecloud/opencda_carla.py:36:8: C0415: Import outside toplevel (carla.Vector3D) (import-outside-toplevel)
ecloud/opencda_carla.py:36:8: E0611: No name 'Vector3D' in module 'carla' (no-name-in-module)
ecloud/opencda_carla.py:11:0: R0903: Too few public methods (1/2) (too-few-public-methods)
ecloud/opencda_carla.py:56:4: W0246: Useless parent or super() delegation in method '__init__' (useless-parent-delegation)
ecloud/opencda_carla.py:57:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
ecloud/opencda_carla.py:69:8: W0621: Redefining name 'Location' from outer scope (line 42) (redefined-outer-name)
ecloud/opencda_carla.py:69:8: W0621: Redefining name 'Vector3D' from outer scope (line 11) (redefined-outer-name)
ecloud/opencda_carla.py:69:8: C0415: Import outside toplevel (carla.Location, carla.Vector3D) (import-outside-toplevel)
ecloud/opencda_carla.py:69:8: E0611: No name 'Location' in module 'carla' (no-name-in-module)
ecloud/opencda_carla.py:69:8: E0611: No name 'Vector3D' in module 'carla' (no-name-in-module)
ecloud/opencda_carla.py:70:16: R1701: Consider merging these isinstance calls to isinstance(location, (Location, Vector3D)) (consider-merging-isinstance)
ecloud/opencda_carla.py:110:8: W0621: Redefining name 'Rotation' from outer scope (line 76) (redefined-outer-name)
ecloud/opencda_carla.py:110:8: C0415: Import outside toplevel (carla.Rotation) (import-outside-toplevel)
ecloud/opencda_carla.py:110:8: E0611: No name 'Rotation' in module 'carla' (no-name-in-module)
ecloud/opencda_carla.py:76:0: R0903: Too few public methods (1/2) (too-few-public-methods)
ecloud/opencda_carla.py:154:8: W0621: Redefining name 'Transform' from outer scope (line 116) (redefined-outer-name)
ecloud/opencda_carla.py:154:8: C0415: Import outside toplevel (carla.Transform) (import-outside-toplevel)
ecloud/opencda_carla.py:154:8: E0611: No name 'Transform' in module 'carla' (no-name-in-module)
ecloud/opencda_carla.py:116:0: R0903: Too few public methods (1/2) (too-few-public-methods)
************* Module eCloud.ecloud.sim_debug_helper
ecloud/sim_debug_helper.py:61:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)
ecloud/sim_debug_helper.py:66:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)
ecloud/sim_debug_helper.py:71:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)
ecloud/sim_debug_helper.py:76:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)
ecloud/sim_debug_helper.py:78:0: C0305: Trailing newlines (trailing-newlines)
ecloud/sim_debug_helper.py:12:0: R0902: Too many instance attributes (11/7) (too-many-instance-attributes)
ecloud/sim_debug_helper.py:30:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
ecloud/sim_debug_helper.py:44:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:47:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:50:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:53:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:56:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:59:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:64:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:69:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/sim_debug_helper.py:74:4: C0116: Missing function or method docstring (missing-function-docstring)
************* Module eCloud.ecloud.client_debug_helper
ecloud/client_debug_helper.py:159:9: W0511: TODO: extend instead of append? or [:] = ? (fixme)
ecloud/client_debug_helper.py:13:0: R0902: Too many instance attributes (11/7) (too-many-instance-attributes)
ecloud/client_debug_helper.py:31:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
************* Module eCloud.ecloud.co_simulation.sumo_integration.carla_simulation
ecloud/co_simulation/sumo_integration/carla_simulation.py:27:0: R0902: Too many instance attributes (8/7) (too-many-instance-attributes)
ecloud/co_simulation/sumo_integration/carla_simulation.py:78:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/co_simulation/sumo_integration/carla_simulation.py:174:25: R1718: Consider using a set comprehension (consider-using-set-comprehension)
************* Module eCloud.ecloud.co_simulation.sumo_integration.sumo_simulation
ecloud/co_simulation/sumo_integration/sumo_simulation.py:35:0: R0903: Too few public methods (0/2) (too-few-public-methods)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:50:0: R0903: Too few public methods (0/2) (too-few-public-methods)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:176:16: C0103: Variable name "tl" doesn't conform to snake_case naming style (invalid-name)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:242:12: C0103: Variable name "tl" doesn't conform to snake_case naming style (invalid-name)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:245:8: R1705: Unnecessary "elif" after "return", remove the leading "el" from "elif" (no-else-return)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:298:11: I1101: Module 'lxml.etree' has no 'parse' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:313:4: R0913: Too many arguments (7/5) (too-many-arguments)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:353:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/co_simulation/sumo_integration/sumo_simulation.py:25:0: C0411: third party import "import lxml.etree as ET" should be placed before "from ecloud.co_simulation.sumo_integration.constants import INVALID_ACTOR_ID" (wrong-import-order)
************* Module eCloud.ecloud.co_simulation.sumo_integration.bridge_helper
ecloud/co_simulation/sumo_integration/bridge_helper.py:42:9: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
ecloud/co_simulation/sumo_integration/bridge_helper.py:127:24: E1101: Instance of 'list' has no 'filter' member (no-member)
ecloud/co_simulation/sumo_integration/bridge_helper.py:142:24: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/co_simulation/sumo_integration/bridge_helper.py:340:8: R1705: Unnecessary "elif" after "return", remove the leading "el" from "elif" (no-else-return)
ecloud/co_simulation/sumo_integration/bridge_helper.py:340:11: R1714: Consider merging these comparisons with 'in' by using 'sumo_tl_state in (SumoSignalState.RED, SumoSignalState.RED_YELLOW)'. Use a set instead if elements are hashable. (consider-using-in)
ecloud/co_simulation/sumo_integration/bridge_helper.py:346:13: R1714: Consider merging these comparisons with 'in' by using 'sumo_tl_state in (SumoSignalState.GREEN, SumoSignalState.GREEN_WITHOUT_PRIORITY)'. Use a set instead if elements are hashable. (consider-using-in)
ecloud/co_simulation/sumo_integration/bridge_helper.py:361:8: R1705: Unnecessary "elif" after "return", remove the leading "el" from "elif" (no-else-return)
************* Module eCloud.ecloud.ecloud_server.ecloud_comms
ecloud/ecloud_server/ecloud_comms.py:114:21: W0511: TODO: confirm if we wantt to do this? (fixme)
ecloud/ecloud_server/ecloud_comms.py:26:0: R0903: Too few public methods (0/2) (too-few-public-methods)
ecloud/ecloud_server/ecloud_comms.py:51:0: R0902: Too many instance attributes (8/7) (too-many-instance-attributes)
ecloud/ecloud_server/ecloud_comms.py:56:4: R0913: Too many arguments (7/5) (too-many-arguments)
ecloud/ecloud_server/ecloud_comms.py:90:16: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:91:16: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:150:16: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:151:16: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:181:15: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/ecloud_server/ecloud_comms.py:166:20: C0103: Variable name "t" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:175:20: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:318:8: C0103: Attribute name "q" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:315:17: C0103: Argument name "q" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:358:33: C0103: Argument name "q" doesn't conform to snake_case naming style (invalid-name)
ecloud/ecloud_server/ecloud_comms.py:369:11: W0718: Catching too general exception Exception (broad-exception-caught)
************* Module eCloud.ecloud.scenario_testing.ecloud_4lane_scenario_dist_config
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:27:0: R0402: Use 'from ecloud.scenario_testing.utils import sim_api' instead (consider-using-from-import)
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:46:0: R0914: Too many local variables (28/15) (too-many-locals)
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:159:11: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:180:23: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:187:19: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:46:0: R0912: Too many branches (23/12) (too-many-branches)
ecloud/scenario_testing/ecloud_4lane_scenario_dist_config.py:46:0: R0915: Too many statements (80/50) (too-many-statements)
************* Module eCloud.ecloud.scenario_testing.ecloud_edge_8_car
ecloud/scenario_testing/ecloud_edge_8_car.py:45:0: C0301: Line too long (137/128) (line-too-long)
ecloud/scenario_testing/ecloud_edge_8_car.py:13:0: R0402: Use 'from ecloud.scenario_testing.utils import sim_api' instead (consider-using-from-import)
ecloud/scenario_testing/ecloud_edge_8_car.py:22:0: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/scenario_testing/ecloud_edge_8_car.py:22:0: R0914: Too many local variables (18/15) (too-many-locals)
ecloud/scenario_testing/ecloud_edge_8_car.py:69:18: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
************* Module eCloud.ecloud.scenario_testing.multi_2lanefree_carla
ecloud/scenario_testing/multi_2lanefree_carla.py:39:0: C0301: Line too long (129/128) (line-too-long)
ecloud/scenario_testing/multi_2lanefree_carla.py:16:0: R0402: Use 'from ecloud.scenario_testing.utils import sim_api' instead (consider-using-from-import)
ecloud/scenario_testing/multi_2lanefree_carla.py:25:0: R0914: Too many local variables (19/15) (too-many-locals)
ecloud/scenario_testing/multi_2lanefree_carla.py:47:26: R1719: The if expression can be replaced with 'test' (simplifiable-if-expression)
ecloud/scenario_testing/multi_2lanefree_carla.py:88:18: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/scenario_testing/multi_2lanefree_carla.py:101:22: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/scenario_testing/multi_2lanefree_carla.py:133:16: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/multi_2lanefree_carla.py:25:0: R0912: Too many branches (17/12) (too-many-branches)
ecloud/scenario_testing/multi_2lanefree_carla.py:25:0: R0915: Too many statements (53/50) (too-many-statements)
************* Module eCloud.ecloud.scenario_testing.ecloud_edge_16_car
ecloud/scenario_testing/ecloud_edge_16_car.py:50:0: C0301: Line too long (137/128) (line-too-long)
ecloud/scenario_testing/ecloud_edge_16_car.py:15:0: R0402: Use 'from ecloud.scenario_testing.utils import sim_api' instead (consider-using-from-import)
ecloud/scenario_testing/ecloud_edge_16_car.py:19:0: W0404: Reimport 'ecloud.scenario_testing.utils.sim_api' (imported line 15) (reimported)
ecloud/scenario_testing/ecloud_edge_16_car.py:19:0: R0402: Use 'from ecloud.scenario_testing.utils import sim_api' instead (consider-using-from-import)
ecloud/scenario_testing/ecloud_edge_16_car.py:28:0: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/scenario_testing/ecloud_edge_16_car.py:28:0: R0914: Too many local variables (18/15) (too-many-locals)
ecloud/scenario_testing/ecloud_edge_16_car.py:74:18: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/scenario_testing/ecloud_edge_16_car.py:10:0: W0611: Unused import logging (unused-import)
ecloud/scenario_testing/ecloud_edge_16_car.py:11:0: W0611: Unused import time (unused-import)
ecloud/scenario_testing/ecloud_edge_16_car.py:16:0: W0611: Unused ecloud.scenario_testing.utils.customized_map_api imported as map_api (unused-import)
************* Module eCloud.ecloud.scenario_testing.evaluations.utils
ecloud/scenario_testing/evaluations/utils.py:31:9: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
************* Module eCloud.ecloud.scenario_testing.evaluations.evaluate_manager
ecloud/scenario_testing/evaluations/evaluate_manager.py:247:33: W0511: TODO: global (fixme)
ecloud/scenario_testing/evaluations/evaluate_manager.py:231:15: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/evaluations/evaluate_manager.py:233:25: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
ecloud/scenario_testing/evaluations/evaluate_manager.py:236:21: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
ecloud/scenario_testing/evaluations/evaluate_manager.py:247:8: C0103: Variable name "PLANER_AGENT_STEPS" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/evaluations/evaluate_manager.py:343:4: R0914: Too many local variables (22/15) (too-many-locals)
ecloud/scenario_testing/evaluations/evaluate_manager.py:423:15: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/evaluations/evaluate_manager.py:425:25: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
ecloud/scenario_testing/evaluations/evaluate_manager.py:343:4: R0912: Too many branches (13/12) (too-many-branches)
ecloud/scenario_testing/evaluations/evaluate_manager.py:343:4: R0915: Too many statements (63/50) (too-many-statements)
ecloud/scenario_testing/evaluations/evaluate_manager.py:428:21: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
************* Module eCloud.ecloud.scenario_testing.utils.cosim_api
ecloud/scenario_testing/utils/cosim_api.py:87:9: W0511: TODO: use yaml file to generate the route file (fixme)
ecloud/scenario_testing/utils/cosim_api.py:225:9: W0511: TODO make sure the tl is synced (fixme)
ecloud/scenario_testing/utils/cosim_api.py:18:0: C0413: Import "import carla" should be placed at the top of the module (wrong-import-position)
ecloud/scenario_testing/utils/cosim_api.py:20:0: C0413: Import "from ecloud.co_simulation.sumo_integration.constants import SPAWN_OFFSET_Z" should be placed at the top of the module (wrong-import-position)
ecloud/scenario_testing/utils/cosim_api.py:21:0: C0413: Import "from ecloud.co_simulation.sumo_integration.bridge_helper import BridgeHelper" should be placed at the top of the module (wrong-import-position)
ecloud/scenario_testing/utils/cosim_api.py:22:0: C0413: Import "from ecloud.co_simulation.sumo_integration.constants import INVALID_ACTOR_ID" should be placed at the top of the module (wrong-import-position)
ecloud/scenario_testing/utils/cosim_api.py:23:0: C0413: Import "from ecloud.co_simulation.sumo_integration.sumo_simulation import SumoSimulation" should be placed at the top of the module (wrong-import-position)
ecloud/scenario_testing/utils/cosim_api.py:25:0: C0413: Import "from ecloud.scenario_testing.utils.sim_api import ScenarioManager" should be placed at the top of the module (wrong-import-position)
ecloud/scenario_testing/utils/cosim_api.py:51:4: R0913: Too many arguments (8/5) (too-many-arguments)
ecloud/scenario_testing/utils/cosim_api.py:51:4: R0914: Too many local variables (17/15) (too-many-locals)
ecloud/scenario_testing/utils/cosim_api.py:58:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
ecloud/scenario_testing/utils/cosim_api.py:88:41: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/scenario_testing/utils/cosim_api.py:116:4: R0914: Too many local variables (17/15) (too-many-locals)
ecloud/scenario_testing/utils/cosim_api.py:166:8: C0206: Consider iterating with .items() (consider-using-dict-items)
ecloud/scenario_testing/utils/cosim_api.py:183:25: R1718: Consider using a set comprehension (consider-using-set-comprehension)
ecloud/scenario_testing/utils/cosim_api.py:213:8: C0206: Consider iterating with .items() (consider-using-dict-items)
ecloud/scenario_testing/utils/cosim_api.py:116:4: R0912: Too many branches (14/12) (too-many-branches)
ecloud/scenario_testing/utils/cosim_api.py:240:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/scenario_testing/utils/cosim_api.py:325:4: W0221: Number of parameters was 2 in 'ScenarioManager.close' and is now 1 in overriding 'CoScenarioManager.close' method (arguments-differ)
************* Module eCloud.ecloud.scenario_testing.utils.sim_api
ecloud/scenario_testing/utils/sim_api.py:159:9: W0511: TODO: move these to EcloudConfig (fixme)
ecloud/scenario_testing/utils/sim_api.py:201:13: W0511: TODO move path to globals (fixme)
ecloud/scenario_testing/utils/sim_api.py:220:37: W0511: TODO: this MUST be a print call and NOT a log call for some reason (fixme)
ecloud/scenario_testing/utils/sim_api.py:290:13: W0511: TODO: support multiple edges... (fixme)
ecloud/scenario_testing/utils/sim_api.py:782:9: W0511: TODO: needs to support multiple edges. (fixme)
ecloud/scenario_testing/utils/sim_api.py:86:0: R0902: Too many instance attributes (24/7) (too-many-instance-attributes)
ecloud/scenario_testing/utils/sim_api.py:143:4: R0913: Too many arguments (11/5) (too-many-arguments)
ecloud/scenario_testing/utils/sim_api.py:143:4: R0914: Too many local variables (19/15) (too-many-locals)
ecloud/scenario_testing/utils/sim_api.py:181:29: R1714: Consider merging these comparisons with 'in' by using 'EcloudConfig.ecloud_ip in ('localhost', EcloudConfig.carla_ip)'. Use a set instead if elements are hashable. (consider-using-in)
ecloud/scenario_testing/utils/sim_api.py:187:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:206:27: R1714: Consider merging these comparisons with 'in' by using 'EcloudConfig.carla_ip in ('localhost', EcloudConfig.ecloud_ip)'. Use a set instead if elements are hashable. (consider-using-in)
ecloud/scenario_testing/utils/sim_api.py:210:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:248:19: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/utils/sim_api.py:248:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:143:4: R0912: Too many branches (26/12) (too-many-branches)
ecloud/scenario_testing/utils/sim_api.py:143:4: R0915: Too many statements (105/50) (too-many-statements)
ecloud/scenario_testing/utils/sim_api.py:202:41: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
ecloud/scenario_testing/utils/sim_api.py:221:33: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
ecloud/scenario_testing/utils/sim_api.py:485:37: C0103: Argument name "tm" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:547:37: C0103: Argument name "tm" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:547:4: R0914: Too many local variables (26/15) (too-many-locals)
ecloud/scenario_testing/utils/sim_api.py:585:16: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:586:20: C0103: Variable name "y" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:663:8: C0103: Variable name "tm" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:697:23: W0718: Catching too general exception Exception (broad-exception-caught)
ecloud/scenario_testing/utils/sim_api.py:697:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
ecloud/scenario_testing/utils/sim_api.py:700:12: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
ecloud/scenario_testing/utils/sim_api.py:762:4: R0913: Too many arguments (6/5) (too-many-arguments)
ecloud/scenario_testing/utils/sim_api.py:919:38: R1714: Consider merging these comparisons with 'in' by using 'EcloudConfig.ecloud_ip in ('localhost', EcloudConfig.carla_ip)'. Use a set instead if elements are hashable. (consider-using-in)
************* Module eCloud.ecloud.scenario_testing.utils.customized_map_api
ecloud/scenario_testing/utils/customized_map_api.py:14:0: C0115: Missing class docstring (missing-class-docstring)
ecloud/scenario_testing/utils/customized_map_api.py:14:0: C0103: Class name "bcolors" doesn't conform to PascalCase naming style (invalid-name)
ecloud/scenario_testing/utils/customized_map_api.py:14:0: R0903: Too few public methods (0/2) (too-few-public-methods)
ecloud/scenario_testing/utils/customized_map_api.py:38:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
ecloud/scenario_testing/utils/customized_map_api.py:39:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
ecloud/scenario_testing/utils/customized_map_api.py:45:14: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
************* Module eCloud.ecloud.scenario_testing.utils.yaml_utils
ecloud/scenario_testing/utils/yaml_utils.py:26:13: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
************* Module eCloud.ecloud.customize.ml_libs.ml_manager
ecloud/customize/ml_libs/ml_manager.py:70:12: C0103: Variable name "x1" doesn't conform to snake_case naming style (invalid-name)
ecloud/customize/ml_libs/ml_manager.py:70:16: C0103: Variable name "y1" doesn't conform to snake_case naming style (invalid-name)
ecloud/customize/ml_libs/ml_manager.py:70:20: C0103: Variable name "x2" doesn't conform to snake_case naming style (invalid-name)
ecloud/customize/ml_libs/ml_manager.py:70:24: C0103: Variable name "y2" doesn't conform to snake_case naming style (invalid-name)
ecloud/customize/ml_libs/ml_manager.py:75:12: E1101: Module 'cv2' has no 'rectangle' member (no-member)
ecloud/customize/ml_libs/ml_manager.py:77:12: E1101: Module 'cv2' has no 'putText' member (no-member)
ecloud/customize/ml_libs/ml_manager.py:78:24: E1101: Module 'cv2' has no 'FONT_HERSHEY_SIMPLEX' member (no-member)
ecloud/customize/ml_libs/ml_manager.py:21:0: R0903: Too few public methods (1/2) (too-few-public-methods)
ecloud/customize/ml_libs/ml_manager.py:94:11: R1719: The if expression can be replaced with 'test' (simplifiable-if-expression)
************* Module eCloud.ecloud.customize.core.sensing.localization.extented_kalman_filter
ecloud/customize/core/sensing/localization/extented_kalman_filter.py:135:4: R0913: Too many arguments (6/5) (too-many-arguments)
************* Module eCloud.ecloud.customize.core.sensing.localization.localization_manager
ecloud/customize/core/sensing/localization/localization_manager.py:32:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
************* Module eCloud.ecloud.core.plan.global_route_planner_dao
ecloud/core/plan/global_route_planner_dao.py:33:4: R0914: Too many local variables (16/15) (too-many-locals)
ecloud/core/plan/global_route_planner_dao.py:57:23: R1735: Consider using '{}' instead of a call to 'dict'. (use-dict-literal)
************* Module eCloud.ecloud.core.plan.local_planner_behavior
ecloud/core/plan/local_planner_behavior.py:447:9: W0511: TODO: hard-coded, need to be tuned (fixme)
ecloud/core/plan/local_planner_behavior.py:41:0: R0902: Too many instance attributes (21/7) (too-many-instance-attributes)
ecloud/core/plan/local_planner_behavior.py:237:4: R0914: Too many local variables (35/15) (too-many-locals)
ecloud/core/plan/local_planner_behavior.py:237:4: R0912: Too many branches (14/12) (too-many-branches)
ecloud/core/plan/local_planner_behavior.py:237:4: R0915: Too many statements (72/50) (too-many-statements)
ecloud/core/plan/local_planner_behavior.py:297:8: W0612: Unused variable 'lane_width' (unused-variable)
ecloud/core/plan/local_planner_behavior.py:405:4: R0914: Too many local variables (16/15) (too-many-locals)
ecloud/core/plan/local_planner_behavior.py:570:4: R0913: Too many arguments (7/5) (too-many-arguments)
ecloud/core/plan/local_planner_behavior.py:616:8: W0201: Attribute '_target_speed' defined outside __init__ (attribute-defined-outside-init)
ecloud/core/plan/local_planner_behavior.py:641:30: W0201: Attribute '_target_speed' defined outside __init__ (attribute-defined-outside-init)
ecloud/core/plan/local_planner_behavior.py:641:8: W0201: Attribute 'target_waypoint' defined outside __init__ (attribute-defined-outside-init)
************* Module eCloud.ecloud.core.plan.collision_check
ecloud/core/plan/collision_check.py:43:4: R0914: Too many local variables (16/15) (too-many-locals)
ecloud/core/plan/collision_check.py:108:15: R1719: The if expression can be replaced with 'test' (simplifiable-if-expression)
ecloud/core/plan/collision_check.py:104:8: W0612: Unused variable 'distance' (unused-variable)
ecloud/core/plan/collision_check.py:110:4: R0913: Too many arguments (6/5) (too-many-arguments)
ecloud/core/plan/collision_check.py:110:4: R0914: Too many local variables (24/15) (too-many-locals)
ecloud/core/plan/collision_check.py:111:49: W0613: Unused argument 'carla_map' (unused-argument)
ecloud/core/plan/collision_check.py:111:60: W0613: Unused argument 'world' (unused-argument)
ecloud/core/plan/collision_check.py:182:4: R0913: Too many arguments (8/5) (too-many-arguments)
ecloud/core/plan/collision_check.py:182:4: R0914: Too many local variables (22/15) (too-many-locals)
ecloud/core/plan/collision_check.py:17:0: W0611: Unused draw_trajetory_points imported from ecloud.core.common.misc (unused-import)
************* Module eCloud.ecloud.core.plan.planer_debug_helper
ecloud/core/plan/planer_debug_helper.py:82:17: W0511: TODO: time-step hardcoded (fixme)
ecloud/core/plan/planer_debug_helper.py:146:9: W0511: TODO: extend instead of append? or [:] = ? (fixme)
ecloud/core/plan/planer_debug_helper.py:63:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/core/plan/planer_debug_helper.py:87:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/core/plan/planer_debug_helper.py:112:24: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/core/plan/planer_debug_helper.py:126:22: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/core/plan/planer_debug_helper.py:129:23: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/core/plan/planer_debug_helper.py:132:23: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/core/plan/planer_debug_helper.py:138:12: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)
ecloud/core/plan/planer_debug_helper.py:139:12: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)
ecloud/core/plan/planer_debug_helper.py:143:4: C0116: Missing function or method docstring (missing-function-docstring)
ecloud/core/plan/planer_debug_helper.py:162:4: C0116: Missing function or method docstring (missing-function-docstring)
************* Module eCloud.ecloud.core.plan.drive_profile_plotting
ecloud/core/plan/drive_profile_plotting.py:25:8: W0612: Unused variable 'i' (unused-variable)
ecloud/core/plan/drive_profile_plotting.py:48:8: W0612: Unused variable 'i' (unused-variable)
ecloud/core/plan/drive_profile_plotting.py:72:8: W0612: Unused variable 'i' (unused-variable)
ecloud/core/plan/drive_profile_plotting.py:94:8: W0612: Unused variable 'i' (unused-variable)
ecloud/core/plan/drive_profile_plotting.py:114:8: W0612: Unused variable 'i' (unused-variable)
ecloud/core/plan/drive_profile_plotting.py:135:8: W0612: Unused variable 'i' (unused-variable)
ecloud/core/plan/drive_profile_plotting.py:212:21: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/core/plan/drive_profile_plotting.py:213:47: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
ecloud/core/plan/drive_profile_plotting.py:171:8: W0613: Unused argument 'ttc_list' (unused-argument)
************* Module eCloud.ecloud.core.plan.global_route_planner
ecloud/core/plan/global_route_planner.py:73:4: R0914: Too many local variables (20/15) (too-many-locals)
ecloud/core/plan/global_route_planner.py:93:17: R1735: Consider using '{}' instead of a call to 'dict'. (use-dict-literal)
ecloud/core/plan/global_route_planner.py:95:26: R1735: Consider using '{}' instead of a call to 'dict'. (use-dict-literal)
ecloud/core/plan/global_route_planner.py:116:43: R1735: Consider using '{}' instead of a call to 'dict'. (use-dict-literal)